keras/backend/tensorflow_backend.py

```diff
@@ -4541,14 +4543,11 @@ def ctc_decode(y_pred, input_length, greedy=True, beam_width=100,
         (decoded, log_prob) = ctc.ctc_beam_search_decoder(
             inputs=y_pred,
             sequence_length=input_length, beam_width=beam_width,
-            top_paths=top_paths, merge_repeated=False)
+            top_paths=top_paths, merge_repeated=merge_repeated)
```

Comment:
This looks like a breaking behavior change. Why does it need to default to True?

Reply:
Oh, seems like a bug, should be `False`. Thanks for the catch! I'll fix that.

Reply:
Fixed. Sorry :)